docs/usage/configuration/specification.md

```diff
@@ -184,6 +184,14 @@ Vector package installs, generally located at `/etc/vector/vector.spec.yml`:
   # * default: true
   current_runtime_only = true
 
+  # The directory used to persist the journal checkpoint position. By default,
+  # the global `data_dir` is used. Please make sure the Vector project has write
+  # permissions to this dir.
+  # 
+  # * optional
+  # * no default
+  data_dir = "/var/lib/vector"
```

Comment:
this sounds like something we should add to the file source too since it has check pointing as well?

Reply:
I got this from the `file` source. I'm not sure what it is we should add to it then.